src/tools/hunspell.cxx

```diff
@@ -2044,6 +2044,13 @@ int main(int argc, char** argv) {
     if (getenv("DICPATH")) {
       path_std_str.append(getenv("DICPATH")).append(PATHSEP);
     }
+    if (getenv("XDG_DATA_DIRS")) {
```

Comment:
While at it, the [basedir spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) should probably be followed to the point (also support `XDG_DATA_HOME` and fall back to `$HOME/.local/share` and `/usr/local/share/:/usr/share/` respectively).